src/resolve/resolved-dns-scope.c

```diff
@@ -477,6 +477,63 @@ static DnsScopeMatch match_link_local_reverse_lookups(const char *domain) {
         return _DNS_SCOPE_MATCH_INVALID;
 }
 
+static DnsScopeMatch match_subnet_reverse_lookups(
+                DnsScope *s,
+                const char *domain,
+                bool exclude_own) {
+        union in_addr_union ia;
+        LinkAddress *a;
+        int f, r;
+
+        assert(s);
+        assert(domain);
+
+        /* Checks whether the specified domain is an reverse address domain (i.e. in the .in-addr.arpa or
+         * .ip6.arpa area), and if so, whether the address matches any of our local subnets. If so, we
+         * consider ourselves relevant for it.
```

Comment:
Could you please clarify this last sentence? As someone not super familiar with resolved, it's not obvious what "it" refers to